setup.cfg

```diff
@@ -13,17 +13,17 @@ markers =
 
 [pep8]
 max-line-length = 99
-ignore = E121,E123,E126,E133,E226,E241,E242,E704
-exclude = build/*,.tox/*,tests/*,ve/*,*/_vendor/*,conda/auxlib/*,conda/progressbar/*
+ignore = E121,E123,E126,E133,E226,E241,E242,E302,E704
```

Comment:
Ahh. `E302`.  Let's leave this here for now. And then remove it at some point in the future.